### ISSUE TITLE
Replace dependency from Forked Zenlink to Original Zenlink once pendulum upgrade polkadot version to polkadot-v0.9.38

### ISSUE THREAD

**Issue by RustNinja** [pendulum-chain/pendulum#222]
Issue related to https://github.com/pendulum-chain/pendulum/issues/217 and https://github.com/pendulum-chain/pendulum/issues/219 
and problem produced by different version of `jsonrpsee` here is fixed issue in forked version : https://github.com/pendulum-chain/Zenlink-DEX-Module/issues/1

Todo list:
once pendulum repo will upgraded to `polkadot-v0.9.38` it is good idea to replace dependency from forked version to original version. because [original version](https://github.com/zenlinkpro/Zenlink-DEX-Module/blob/master/zenlink-protocol/rpc/Cargo.toml#L16) has a proper dependency of `jsonrpsee` that does not force us to use a forked version.

`https://github.com/pendulum-chain/Zenlink-DEX-Module`
to 
`https://github.com/zenlinkpro/Zenlink-DEX-Module`

**Comment by ebma** [comment]
@vadaynujra this is never ever 'required'. It's just nice to have and acts as a reminder of why we had to use a forked version of the Zenlink pallet basically. We might be able to close it in a few weeks, once Zenlink updates their code to `0.9.40` (as we decided to skip upgrading to `0.9.38` and immediately reached for `0.9.40` but this now makes us rely on the fork again).

**Comment by TorstenStueber** [comment]
@ebma @gianfra-t  could we actually work on this now with the upgrade to Polkadot v0.9.42?

**Comment by gianfra-t** [comment]
Yes, actually in [the upgrade pr](https://github.com/pendulum-chain/pendulum/pull/382) we are already using the `zenlinkpro` repo.

**Comment by TorstenStueber** [comment]
Great. So then we can close this ticket once the other PR is merged.
